telemetry/client.go

```diff
@@ -340,6 +371,21 @@ func (c *Client) ProcessPeerCount(peerCount PeerCount) *json.RawMessage {
 	return &jsonRawMessage
 }
 
+func (c *Client) ProcessPeerConnFailure(peerConnFailure PeerConnFailure) *json.RawMessage {
+	postBody := map[string]interface{}{
+		"failedPeerId":  peerConnFailure.FailedPeerId,
+		"failureCount":  peerConnFailure.FailureCount,
+		"nodeName":      c.nodeName,
+		"nodeKeyUID":    c.keyUID,
+		"peerId":        c.peerId,
+		"statusVersion": c.version,
+		"timestamp":     time.Now().Unix(),
+	}
+	body, _ := json.Marshal(postBody)
+	jsonRawMessage := json.RawMessage(body)
```

Comment:
We marshal and unmarshal here 🤔. And then we marshal again here:
https://github.com/status-im/status-go/blob/780e3e55f45c03b06b783e57056940b2a0ebd218/telemetry/client.go#L230

It would be nice to reduce these procedures. We could change `Client.TelemetryData` type to `interface{}` and pass the `postBody` pointer directly. And then just call `json.Marshal` on it.

I guess it can be done out of this PR ofc.